core/arch/aarch64/aarch64.asm

```diff
@@ -226,16 +226,42 @@ save_priv_mcontext_helper:
         str      w2, [x0, #(16 * ARG_SZ*2 + 12)]
         str      w3, [x0, #(16 * ARG_SZ*2 + 16)]
         add      x4, x0, #simd_OFFSET
-        st1      {v0.2d-v3.2d}, [x4], #64
-        st1      {v4.2d-v7.2d}, [x4], #64
-        st1      {v8.2d-v11.2d}, [x4], #64
-        st1      {v12.2d-v15.2d}, [x4], #64
-        st1      {v16.2d-v19.2d}, [x4], #64
-        st1      {v20.2d-v23.2d}, [x4], #64
-        st1      {v24.2d-v27.2d}, [x4], #64
-        st1      {v28.2d-v31.2d}, [x4], #64
-        /* TODO i#5365: Save Z/P regs as well? Will require runtime check of
-         * ID_AA64PFR0_EL1 for FEAT_SVE.
+
+        /* Registers Q0-Q31 map directly to registers V0-V31. */
```

Comment:
@derekbruening Do you know if any existing test provides coverage for this change? Probably not because it didn't fail on/after #6725 (but then so didn't signalNNN1 on Github Actions). Maybe we need to port detach_state to AArch64 and add more state verifications to it?

Reply:
client.cleancall has some mcontext field checks: but it looks like it was never ported to aarchxxx.  Yes, getting more testing parity on aarchxx would be good.

Reply:
But let's not block this PR on that I assume?

Reply:
Agreed.

Reply:
I've created https://github.com/DynamoRIO/dynamorio/issues/6759 to improve coverage. If you have any other suggestions of x86 tests not ported to AArch64 which will catch bugs like this, please add them to that issue.

Reply:
Thanks for creating #6759!